src/frontc/frontc.ml

```diff
@@ -202,7 +203,9 @@ and parse_to_cabs_inner (fname : string) =
       ignore (E.log "Parsing error");
       Clexer.finish ();
       close_output ();
-      raise (ParseError("Parse error"))
+      (* raise (ParseError("Parse error")) *)
+      let backtrace = Printexc.get_raw_backtrace () in
+      Printexc.raise_with_backtrace (ParseError("Parse error")) backtrace (* re-raise with captured inner backtrace *)
```

Comment:
Idk, do we need to support 4.04.2?
I would just drop it or comment it out again instead of adding a dependency for it.
It just makes it easier to debug what's going on on error.